server/src/test/java/io/pravega/schemaregistry/storage/client/WireCommandMock.java

```diff
@@ -37,10 +36,7 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.stream.Collectors;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyBoolean;
-import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
nit: Should be single line imports instead. Please add that check to Checkstyle as well.